packages/web/src/pages/sign-in-page/SignInPageDesktop.tsx

```diff
@@ -76,7 +80,10 @@ export const SignInPageDesktop = () => {
             text={messages.createAccount}
           />{' '}
         </LeftContentContainer>
-      </PageWithAudiusValues>
+        <ArtworkContainer isDesktop>
```

Comment:
can these just do `useMedia` internally?

Reply:
Sure, down to update all the internal `isDesktop`s